prov/efa/src/efa_prov.c

```diff
@@ -109,6 +109,9 @@ static int efa_util_prov_initialize()
 			continue;
 		}
 
+		/* This function must be called after efa_hmem_info_initialize() */
+		efa_prov_info_set_hmem_flags(prov_info_direct, FI_EP_RDM);
```

Comment:
probably nit: If we're only going to call this function here, can we rename it to `efa_prov_info_direct_set_hmem_flags` and remove the `ep_type` argument?

Reply:
Updated.